Podfile

```diff
@@ -1,19 +1,22 @@
 workspace 'Stitch.xcworkspace'
```

Comment:
[q] Should this be committed and merged to master prior to the rest of this review, bypassing sync branch?

Reply:
We had to change dependencies for this PR.